docs/marionette.region.md

```diff
@@ -463,3 +487,60 @@ var MyView = Mn.View.extend({
   }
 });
 ```
+
+## Configure How To Remove View
+
+Override the region's `removeView` method to change how and when the view is destroyed / removed
+from the DOM. This method receives one parameter - the view to remove.
+
+The default implementation of `removeView` is:
+
+```javascript
+var Mn = require('backbone.marionette');
+
+Mn.Region.prototype.removeView = function(view){
+  this.destroyView(view);
+}
+```
+
+> `destroyView` method destroys the view taking into consideration if is
+> a Marionette.View descendant or vanilla Backbone view. It can be replaced
+> by a `view.destroy()` call if is ensured that view descends from Marionette.View   
+
+This example will animate with a fade effect showing and hiding the view:
+
+```javascript
+var Mn = require('backbone.marionette');
+
+var AnimatedRegion = Mn.Region.extend({
+  attachHtml(view) {    
+    view.$el
+      .css({display: 'none'})
+      .appendTo(this.$el);      
+    if (!this.isSwappingView()) view.$el.fadeIn('slow');
+  },
+  
+  removeView(view) {
+    view.$el.fadeOut('slow', () => {
```

Comment:
Can you change this to ES5 functions please?

```js
var self = this;
view.$el.fadeOut('slow', function() {
  self.destroyView(view);
  self.currentView.$el.fadeIn('slow');
});
```

I prefer the `() =>` syntax as well but this will prevent us having to explain the syntax to newer developers.